pepsico/comparison.py

```diff
@@ -1,14 +1,66 @@
 import xarray as xr
-    
-# defining the path to file  
-ds_zarr = '../data/remic/mydatafiles/zarr/test/'
+import os
+#from pathlib import Path 
```

Comment:
you can remove if not using. We use comments only that help code reading. Actual unused code can be deleted rather than commented: that's what git version-control is for (to retrieve potentially past code).

Reply:
Ok makes sense